styles/index.css

```diff
@@ -0,0 +1,173 @@
+body {
+    background-image: url('/assets/weather-background.jpg');
+    opacity: 70;
+}
```

Comment:
to fit your image across the page and not repeat you can add
` background-size: cover;`
I attached two screenshots so you can see the difference
<img width="1247" alt="Screen Shot 2022-06-18 at 10 04 37 AM" src="https://user-images.githubusercontent.com/60710670/174442087-6f7db818-ae58-4c80-988e-2f3f9dd16902.png">

<img width="1137" alt="Screen Shot 2022-06-18 at 10 04 13 AM" src="https://user-images.githubusercontent.com/60710670/174442093-36d6d918-10ce-429b-be7f-2ce3d1fcc6ce.png">